src/APIResult.php

```diff
@@ -12,6 +12,12 @@ public function __construct($res)
 	
 	public function isError()
 	{
+		// sendinvoicebyemail method returns NULL on successful request
+	        // TODO I think its time to start checking status code as well
```

Comment:
Send function that makes the CURL request checks for the status code anything that is not 200 yields and error. I do agree that it is not very graceful solution. So it would probably make sense to include the status code in the APIResult and have it be part of the APIResult, not just the body of the request.